tests/accounts/base.py

```diff
@@ -0,0 +1,81 @@
+import os
+import uuid
+
+from demands import HTTPServiceError
+from mock import Mock
+from unittest2 import TestCase, skip
+from yoconfig import configure
+from yoconfigurator.base import read_config
+
+from sitewit.services import SitewitService
+
+
+class BaseTestCase(TestCase):
```

Comment:
@blaix said that the code smells when `Base` is used in the name of the base class.